easybuild/easyconfigs/g/GCCcore/GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'EB_GCC'
+
+name = 'GCCcore'
+version = '6.4.0'
+
+homepage = 'http://gcc.gnu.org/'
+description = """The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
+ as well as libraries for these languages (libstdc++, libgcj,...)."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+mpfr_version = '3.1.5'
+
+source_urls = [
+    'http://ftpmirror.gnu.org/gnu/gcc/gcc-%(version)s',  # GCC auto-resolving HTTP mirror
+    'http://ftpmirror.gnu.org/gnu/gmp',  # idem for GMP
+    'http://ftpmirror.gnu.org/gnu/mpfr',  # idem for MPFR
+    'http://www.multiprecision.org/mpc/download',  # MPC official
+    'ftp://gcc.gnu.org/pub/gcc/infrastructure/',  # GCC dependencies
+    'http://gcc.cybermirror.org/infrastructure/',  # HTTP mirror for GCC dependencies
+    'http://isl.gforge.inria.fr/',  # original HTTP source for ISL
+]
+
+sources = [
+    'gcc-%(version)s-RC-20170628.tar.gz',  # FIXME 'gcc-%(version)s.tar.bz2',
```

Comment:
currently testing with release candidate for GCC 6.4.0, this needs to be changed before merging